src/main/java/uk/gov/hmcts/darts/common/entity/SecurityGroupEntity.java

```diff
@@ -68,4 +68,10 @@ public class SecurityGroupEntity {
         inverseJoinColumns = {@JoinColumn(name = "cth_id")})
     private Set<CourthouseEntity> courthouseEntities = new LinkedHashSet<>();
 
+    @ManyToMany
```

Comment:
UserAccountEntity_.SECURITY_GROUP_ENTITIES
 `@ManyToMany(mappedBy = SECURITY_GROUP_ENTITIES)`
 So no need to have JoinTable annotation on child side.